src/ring_theory/witt_vector/basic.lean

```diff
@@ -246,6 +246,12 @@ lemma ghost_component_apply (n : ℕ) (x : 𝕎 R) : ghost_component n x = aeval
 
 @[simp] lemma ghost_map_apply (x : 𝕎 R) (n : ℕ) : ghost_map x n = ghost_component n x := rfl
 
+-- TODO: upgrade `x ↦ x.coeff 0` to a ring hom
+lemma coeff_mul_zero (x y : 𝕎 R) : (x * y).coeff 0 = x.coeff 0 * y.coeff 0 :=
+by simp [mul_coeff, peval]
```

Comment:
This lemma probably makes more sense in `defs.lean` just below `neg_coeff`

Reply:
... and shoud maybe be called `mul_coeff_zero` to match `mul_coeff`, even though that name seems backwards.